catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/teams/RoleResourceTest.java

```diff
@@ -205,7 +213,7 @@ public void validateGetWithDifferentFields(Role expectedRole, boolean byName) th
     validateRole(role, expectedRole.getDescription(), expectedRole.getDisplayName(), updatedBy);
 
     // .../roles?fields=policy,users
-    String fields = "policy,users";
+    String fields = "policy,teams,users";
```

Comment:
Can we add validation for teams as well?

```
TestUtils.validateEntityReference(role.getTeams());
```

Reply:
Added. Thanks for pointing this out.